postgres_exporter/common/queries_nodemx.yml

```diff
@@ -112,7 +133,11 @@ ccp_nodemx_cpucfs:
         description: "the length of a period (in microseconds)"
 
 ccp_nodemx_cpuacct:
-  query: "SELECT monitor.cgroup_scalar_bigint('cpuacct.usage') as usage, clock_timestamp() as usage_ts"
+  query: "SELECT CASE WHEN monitor.cgroup_mode() = 'legacy'
```

Comment:
🌱 I'm curious why this file is using YAML flow and double quotes `"` for these multi-line queries. The block style with `|` or `|-` seems better suited. 🤔 Even for single-line queries, we wouldn't have to worry about escape sequences.

https://yaml-multiline.info/